aio/tools/transforms/templates/api/lib/memberHelpers.html

```diff
@@ -234,6 +234,12 @@ <h2>{$ headingText $}</h2>
           {$ property.shortDescription | marked | trim $}{% endif %}
           {$ (property.description or property.constructorParamDoc.description) | marked $}
 
+          {%- if property.deprecated !== undefined %}
+          <p class="deprecated">
+            {$ ('**Deprecated** ' + property.deprecated) | marked | trim $}
+          </p>
```

Comment:
The `marked` pipe/filter already adds `<p>` elements around the text, so we should not add them in manually. I am not sure if the `.deprecated` CSS class has any effect anyway.

```suggestion
          {$ ('**Deprecated** ' + property.deprecated) | marked | trim $}
```

Reply:
With and without the extra `<p>`:

<img width="823" alt="Screenshot 2021-09-24 at 14 54 18" src="https://user-images.githubusercontent.com/15655/134686238-e85b1530-a007-4752-bac3-c4d5ac4df5ff.png">
<img width="834" alt="Screenshot 2021-09-24 at 14 54 11" src="https://user-images.githubusercontent.com/15655/134686241-5e351e5e-6838-4992-a507-ad9fd8737a59.png">

Reply:
By looking at the Sass files, the `.deprecated` class doesn't seem to have any effect here, so let's remove it :+1: